Kitodo/src/main/java/de/sub/goobi/converter/URIConverter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package de.sub.goobi.converter;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.faces.component.UIComponent;
+import javax.faces.context.FacesContext;
+import javax.faces.convert.Converter;
+import javax.faces.convert.ConverterException;
+import javax.faces.convert.FacesConverter;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@FacesConverter("URIConverter")
+public class URIConverter implements Converter {
+    public static final String CONVERTER_ID = "URIConverter";
+    private static final Logger logger = LogManager.getLogger(DocketConverter.class);
```

Comment:
Wrong name for logging instance: please change it from `DocketConverter.class` to `URIConverter.class`